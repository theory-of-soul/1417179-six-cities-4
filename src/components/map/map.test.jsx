import React from "react";
import renderer from 'react-test-renderer';
import Map from "./map";

const city = [52.38333, 4.9];
const markersCoordinates = [[52.3909553943508, 4.85309666406198], [52.369553943508, 4.85309666406198]];
const options = {
  createNodeMock: (element) => {
    if (element.props.id === `map`) {
      return document.createElement(`div`);
    }
    return null;
  }
};

describe(`Map component snapshot tests`, () => {
  it(`Map component render map container`, () => {
    const tree = renderer
      .create(<Map city={city} markersCoordinates={markersCoordinates}/>, options).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
