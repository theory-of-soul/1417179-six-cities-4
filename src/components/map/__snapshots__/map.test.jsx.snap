// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component snapshot tests Map component render map container 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
